packages/palette/src/elements/Toggle/Toggle.story.tsx

```diff
@@ -17,6 +18,13 @@ const SecondaryAction = () => {
   )
 }
 
+const labelComponent = (
```

Comment:
#superminor - Since this is a react component we should always use `TitleCase`. (The real-world requirement here is due to the fact that react hooks don't work  inside of lowercase-named components, amongst other things.) Also no need to include "component" in var name, can keep it simple like: `const Label = ...`

Reply:
This is the kind of thing I'd just inline. Typically when I see this I think someone just wants to extract a component; in this case nothing complex is happening to warrant the assignment.

Reply:
(We can inline it, but the comment above is more of a rule of thumb when writing in-general react code)

Reply:
Moved it in-line! But also thanks for calling out b/c I often forget when I'm defining a small React component within a file vs. just a regular variable.